DetectorDescription/DDCMS/src/DDNamespace.cc

```diff
@@ -218,6 +266,19 @@ dd4hep::Volume DDNamespace::volume(const string& name, bool exc) const {
   return nullptr;
 }
 
+dd4hep::Volume* DDNamespace::getVolPtr(const string& name) const {
```

Comment:
@cvuosalo - please, use `ns.volume(name)`

Reply:
A polymorphic pointer is needed. `volumes` seems to contain pure volumes.